nodejs/eks/nodes/nodegroup.ts

```diff
@@ -1933,6 +1953,39 @@ export function createManagedNodeGroup(
         });
     }
 
+    // Create a placement group if EFA support is enabled
+    const createPlacementGroup = args.enableEfaSupport === true;
+    const placementGroup = createPlacementGroup
+        ? new aws.ec2.PlacementGroup(
+              name,
+              {
+                  strategy: "cluster",
+                  tags: args.tags,
+              },
+              { parent, provider },
+          )
+        : undefined;
+
+    // Require users to be explicit about which AZs they want to use when enabling EFA support or creating a placement group.
```

Comment:
I'm curious what y'all think about this. I added my reasoning for this to the PR description as well. This prevents a major foot gun

Reply:
I agree

Reply:
Have to trust you on this one, do not have a good enough mental picture of the implications here.